src/FetchInterceptor.js

```diff
@@ -125,9 +126,7 @@ export default class FetchInterceptor {
 
   isConfigValid() {
     return this.config.shouldIntercept &&
-      this.config.authorizeRequest &&
-      this.config.createAccessTokenRequest &&
-      this.config.parseAccessToken;
+      this.config.authorizeRequest;
```

Comment:
Is this the same config that is already validated in the `AccessTokenProvider`? If yes, can we validate it only in one place. If not, maybe we should rename those two configs so that each one has a more specific name...